api/src/index.ts

```diff
@@ -65,6 +188,21 @@ app.get('/', (c) => {
           path: '/tools/config/:wa/:version',
           methods: ['GET'],
           description: 'Get tools configuration for a wallet address'
+        },
+        {
+          path: '/tools/payment/quote',
+          methods: ['POST'],
+          description: 'Create payment quote'
+        },
+        {
+          path: '/tools/payment/grant',
+          methods: ['POST'],
+          description: 'Create outgoing payment grant'
+        },
+        {
+          path: '/tools/payment/finalize',
+          methods: ['POST'],
+          description: 'Finish payment process'
```

Comment:
If this is just meant for us, maybe we can use 
```ts
      endpoints: app.routes.filter((e) => e.method !== 'ALL'),
```

Although we lose the "description".